### PR TITLE
chore: add prettier check on pr lint

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "format": "prettier --write --config .prettierrc.json --ignore-path .prettierignore \"{src,tests}/**/*.ts\" \"*.{js,json}\"",
     "lint:fix": "eslint \"src/**/*.ts\" \"tests/**/*.ts\" --fix",
     "lint": "eslint \"src/**/*.ts\" \"tests/**/*.ts\"",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "format-check": "bash ../../.github/scripts/lint-pr.sh $npm_config_source 14"
```
Comment (LongOddCode) [inline packages/cli/package.json:31]: maybe it's better not to depends on external files

Comment (wenytang-ms) [reply packages/cli/package.json:31]: it's better to extract the bash script as a common file for easy understanding or fix

Comment (wenytang-ms) [reply packages/cli/package.json:31]: Follow-up with eslint check, you can also write the code into the common file

Comment (wenytang-ms) [reply packages/cli/package.json]: thank you! 
code updated.

**Review by LongOddCode — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]
approve for cli part

**Pull request merged**